### ISSUE TITLE
Make first round of copy changes on website

### ISSUE THREAD

**Issue by pjatx** [Corona-Trace/coronatrace.org#14]
Some changes have been suggested by Tarun, Laura, and Sarah and folded into the attached Google document. 

### Acceptance Criteria: 
- [x] Finalize google doc so it's clear what needs to be added, updated, or remove
- [x] Make updates in contentful
- [x] Create new issues for each new section so they can be designed, built, and implemented into the CMS

### Google Doc
https://docs.google.com/document/d/1EPkk4RD_NetW_d3BmIGPwnFnIN6ZkOCQM34hUlHW8HQ/edit#heading=h.dalul3o2d77u

**Comment by pjatx** [comment]
Currently, I'm blocked by having content folks give their input on the above doc.
